public_html/templates/index.html

```diff
@@ -7,116 +7,147 @@
+									title="The edit was a copyright violation and has been reverted"
+									disabled
+								>
+									Page fixed
+								</button>
+								<button id="{{ 'danger' ~ row.ithenticate_id }}"
+									data-id="{{ row.ithenticate_id }}" data-status="false"
+									class="btn btn-danger-clicked btn-block js-save-state"
+									title="The edit is a false positive, nothing needs to be done"
+								>
+									No action needed
+								</button>
+							{% elseif row.status == NULL %}
+								<button id="{{ 'success' ~ row.ithenticate_id }}"
+									data-id="{{ row.ithenticate_id }}" data-status="fixed"
+									class="btn btn-success btn-outline btn-block js-save-state"
+									title="The edit was a copyright violation and has been reverted"
+								>
+									Page fixed
+								</button>
+								<button id="{{ 'danger' ~ row.ithenticate_id }}"
+									data-id="{{ row.ithenticate_id }}" data-status="false"
+									class="btn btn-danger btn-block js-save-state"
+									title="The edit is a false positive, nothing needs to be done"
+								>
+									No action needed
+								</button>
+							{% endif %}
+							{{ macros.reviewer_node( row ) }}
+						</div>
 					</div>
 				</div>
-			</div>
-			<div class="ithenticate-id">{{ row.ithenticate_id }}</div>
-			<div class="compare-links-container">
-				<a class="btn btn-xs btn-primary compare-button" href='{{ row.turnitin_report }}' target="_blank">
-					<span class="glyphicon glyphicon-new-window"></span>
-					Turnitin report
-				</a><br>
-				{% if row.copyvios is not empty %}
-					{% for copy in row.copyvios %}
-						<div class="compare-div">
-							<button class="btn btn-xs btn-primary dropdown-toggle compare-button"
-							        onclick="toggleComparePane( {{ row.ithenticate_id }}, {{ loop.index }}, '{{ copy }}', {{ row.diff }} )">
-								<span class="glyphicon glyphicon-chevron-down" aria-hidden="true"></span>
-								Compare
-							</button>
-							<a href="{{ copy }}" target="_blank">{{ copy }}</a><br/>
-							<div class="compare-pane" id="{{ 'comp' ~ row.ithenticate_id ~ '-' ~ loop.index }}">
-								<div class="compare-pane-left">
-									<span class="text-muted">Loading article revision...</span>
-								</div>
-								<div class="compare-pane-right">
-									<span class="text-muted">Loading copyvio source...</span>
+				<div class="ithenticate-id">{{ row.ithenticate_id }}</div>
+				<div class="compare-links-container">
+					<a class="btn btn-xs btn-primary compare-button" href='{{ row.turnitin_report }}' target="_blank">
+						<span class="glyphicon glyphicon-new-window"></span>
+						Turnitin report
+					</a><br>
+					{% if row.copyvios is not empty %}
+						{% for copy in row.copyvios %}
+							<div class="compare-div">
+								<button class="btn btn-xs btn-primary dropdown-toggle compare-button"
+												onclick="toggleComparePane( {{ row.ithenticate_id }}, {{ loop.index }}, '{{ copy }}', {{ row.diff }} )">
```

Comment:
I am guessing you didn't change this the way you changed others because it's trickier? Or did you just miss it? :)